src/mpi/coll/transports/gentran/gentran_impl.c

```diff
@@ -10,7 +10,7 @@
     - name        : MPIR_CVAR_PROGRESS_MAX_COLLS
       category    : COLLECTIVE
       type        : int
-      default     : 8
+      default     : 0
```

Comment:
Please add in the commit message its rationale -- performance impact?

Reply:
done, but moved to a new PR at: https://github.com/pmodels/mpich/pull/5506